python/ray/data/tests/test_numpy_support.py

```diff
@@ -27,22 +28,31 @@ def assert_structure_equals(a, b):
     assert a.dtype == b.dtype
     assert a.shape == b.shape
     for i in range(len(a)):
-        assert np.array_equiv(a[i], b[i]), (i, a, b)
+        assert np.array_equal(a[i], b[i]), (i, a[i], b[i])
 
 
-def test_list_of_scalars(ray_start_regular_shared):
+def test_list_of_scalars(ray_start_regular_shared, restore_data_context):
+    # Disable (automatic) fallback to `ArrowPythonObjectType` extension type
+    DataContext.get_current().enable_fallback_to_arrow_object_ext_type = False
```

Comment:
consider defining a fixture, as it's repeated many times in this file.

Reply:
Could be slightly more concise. As we could define cut the duplicated comment and `DataContext.get_current()`, and also combine `restore_data_context` in this fixture. 
But I don't have strong preference, if you prefer the current way.